src/boot/app/app-loader-mounter.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Zextras <https://www.zextras.com>
+ *
```

Comment:
In one file I have seen Year **2022** .Which one is correct 2021 or 2022 ?

Reply:
both, it's the year of creation of the file